src/components/Articles/PostComment.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import { postComment } from "../../assets/api-calls";
+import FloatingLabel from "react-bootstrap/FloatingLabel";
+import Form from "react-bootstrap/Form";
+import "./PostComment.css";
+import Button from "react-bootstrap/Button";
+
+function PostComment({ setComments, article_id }) {
+  const [newComment, setNewComment] = useState("");
+  const [loading, isLoading] = useState(false);
+  const [commentCount, setCommentCount] = useState(0);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    postComment(article_id, { username: "tickle122", body: newComment })
+      .then((newCommentFromAPI) => {
+        isLoading(true);
```

Comment:
is this the right place for loading to be true? at this point the api call has already happened.